### PR TITLE
added i3 with feh support

### PR DESCRIPTION
**Pull request by DeaSTL**
Get wrecked.

### CODE REVIEW

**Comment by DeaSTL** [comment]
image

File: internal/errprefix/err_prefix.go [hunk 1]
```diff
@@ -11,6 +11,23 @@ var prefixes = []string{
 	"segmentation fault (core dumped)... JUST KIDDING, but really the program failed",
 	"Damn bro, I'm sorry but: ",
 	"心からのお詫び: ",
+  "I'm sorry, I relly fucked up here : ",
```
Comment (npmaile) [inline internal/errprefix/err_prefix.go]: delet swearing

File: pkg/papesetter/linux.go [hunk 2]
```diff
@@ -20,12 +20,15 @@ func SetPape(s string) error {
 
 func getDesktop() DE {
 	switch os.Getenv("XDG_CURRENT_DESKTOP") {
```
Comment (npmaile) [inline pkg/papesetter/linux.go]: run `go fmt`

File: pkg/papesetter/de/i3.go [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+import (
+	"os/exec"
+  "fmt"
+  "os"
+  "strings"
+)
+
+type I3 struct{}
+
```
Comment (npmaile) [inline pkg/papesetter/de/i3.go:11]: run `go fmt`

**Review by npmaile — CHANGES_REQUESTED** [review]

**Review by npmaile — APPROVED** [review]

**Pull request merged**